### PR TITLE
remove redundant logic

### LINKED ISSUE #2425: [BUG] Error encountered while developing on Mac
error message:
```
1.670945472725842e+09	ERROR	alluxioctl.AlluxioRuntime	controllers/runtime_controller.go:238	SetupMaster	{"alluxioruntime": "default/demo", "engine": "alluxio", "id": "default-demo", "error": "failed to parse cluster domain from resolv.conf"}
```

https://github.com/fluid-cloudnative/fluid/blob/master/pkg/common/cluster_domain.go#L13

This logic implies that the local `/etc/resolv.conf` must contain the relevant information. Is this by design?

### PR DESCRIPTION
**Pull request by allenhaozi**
causing local development issues
Signed-off-by: mahao <allenhaozi@gmail.com>

### Ⅰ. Describe what this PR does
remove redundant logic
```
clusterDomain, err := common.GetClusterDomain()
	if err != nil {
		return err
	}
	value.ClusterDomain = clusterDomain
```
causing local development issues

### Ⅱ. Does this pull request fix one issue?

fixes #2425

### CODE REVIEW

File: pkg/ddc/alluxio/transform.go [hunk 1]
```diff
@@ -242,12 +242,6 @@ func (e *AlluxioEngine) transformCommonPart(runtime *datav1alpha1.AlluxioRuntime
 
 	e.transformShortCircuit(runtimeInfo, value)
 
-	clusterDomain, err := common.GetClusterDomain()
-	if err != nil {
-		return err
-	}
-	value.ClusterDomain = clusterDomain
```
Comment (cheyang) [inline pkg/ddc/alluxio/transform.go:249]: Do you want to still keep attribute `ClusterDomain`?

Comment (allenhaozi) [reply pkg/ddc/alluxio/transform.go:249]: If this variable is not used, I recommend deleting it

Comment (cheyang) [reply pkg/ddc/alluxio/transform.go:249]: So please delete ClusterDomain in this PR. Thanks.

**Review by cheyang — CHANGES_REQUESTED** [review]

**Comment by cheyang** [comment]
@allenhaozi Thanks very much for your contribution~!

**Pull request merged**